### DISCUSSION TITLE
Paar Fragen vom Anfänger

### DISCUSSION THREAD

**Discussion by OpenShorty** [tbnobody/OpenDTU#991] [Q&A] [answered]
Hallo ich bin ganz neu in diesem Bereich und habe mir meine eigene OpenDTU zusammen gebaut. Erst einmal vielen Dank an alle die diese Projekt möglich gemacht haben. Es ist wirklich erstaunlich was alles so möglich ist. Der Zusammenbau der DTU war grundsätzlich für mich als Elektroniker nicht sehr schwer. Dank allen die hier Infos weiter geben und auch die Anleitungen die hier zu finden sind geht das Flashen und installieren leicht von der Hand. Vielen Dank dafür.

Nun habe ich meine DTU in Betrieb und alles läuft wie es sein soll. Doch wenn man neu ist kommen dann die ein oder andere Frage auf. Ich habe gesucht aber leider keine passenden Antworten gefunden. Ich hoffe ihr könnt mir helfen.

1. Farge:
Welche Bedeutungen haben die Farben mittig der Live-Daten, dort wo man den Wechselrichter z.b. ausschalten kann. ich habe drei Farben beobachtet, Blau (normaler Betrieb?), rot (keine Verbindung WR ausgeschaltet) und auch orange habe ich gesehen. gibt es eine Beschreibung der Farben?

2. Frage:
sehe ich es richtig das die Funktion mit dem Deepsleep so nicht in der Version verfügbar ist? Sprich ich habe heute die v23.6.1 von tbnobody installiert und ich möchte den Deepsleep dann muss ich eine andere Firmware aufspielen? 

3. Frage: 
ist die Option mit den Logo auf dem Display abgeschafft worden?

4. Frage:
ich habe zwischenzeitlich in den Ereignissen zwei komische Fehlermeldungen.
Fehlernummer 2 DTU Command failed
Fehlernummer 12 Unknown

hat da jemand eine Idee für mich?

So entschuldigt meine Fragestunde, das wars für erste, ich hoffe Ihr könnt mir helfen.

LG aus Hamburg

**Comment by OpenShorty** [comment]
keiner eine Idee zu meinen Fragen?

**Comment by DPO99** [comment, marked as answer]
> 1. Farge:
>    Welche Bedeutungen haben die Farben mittig der Live-Daten, dort wo man den Wechselrichter z.b. ausschalten kann. ich habe drei Farben beobachtet, Blau (normaler Betrieb?), rot (keine Verbindung WR ausgeschaltet) und auch orange habe ich gesehen. gibt es eine Beschreibung der Farben?

Gelb/Orange wäre dann noch, WR ist erreichbar aber ausgeschalten oder aber nach z.B. einem Neustart der DTU, wenn es noch keine Verbindung zum WR gab. Eine sep. Beschreibung gibts m.E. nicht.

> 2. Frage:
>    sehe ich es richtig das die Funktion mit dem Deepsleep so nicht in der Version verfügbar ist? Sprich ich habe heute die v23.6.1 von tbnobody installiert und ich möchte den Deepsleep dann muss ich eine andere Firmware aufspielen?

Wozu bräuchtest Du denn diese Funktion? Strom lässt sich so nicht sparen, da der ESP sowieso kaum was braucht.

> 3. Frage:
>    ist die Option mit den Logo auf dem Display abgeschafft worden?

Aktuell gibts da m.E. kein Logo, wäre ja aber auch überflüssig/platzverschwendung.

> 4. Frage:
>    ich habe zwischenzeitlich in den Ereignissen zwei komische Fehlermeldungen.
>    Fehlernummer 2 DTU Command failed
>    Fehlernummer 12 Unknown

Naja DTU Command failed heißt eben, dass es einen Kommunikationsfehler gab (einzeln nicht wild). Fehler 12 ist unbekannt, da unbekannt (nicht klar, was der Fehler bedeutet).

**Comment by tbnobody** [comment]
Blau: Alles Normal
Gelb: Erreichbar aber keine Produktion
Rot: Nicht Erreichbar

**Comment by OpenShorty** [comment]
Vielen Dank für eure Antworten.
Teilweise waren die Fragen zu den Funktionen wie Deepsleep oder Logo auf dem Display nicht das ich sie brauche oder vermisse.
Ich habe mich nur mit dem Thema beschäftigt und es hat mich interessiert.
